_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,346 @@
+3. Leverage the functionalities provided by the selected VCS as much as possible, for instance:
+    - File hosting (source code, scripts, system/database configurations...);
+    - Issues (incl. milestones and Kanban);
+    - Wiki (documentation);
+    - Build and Test Automation (e.g., security-related); and
+    - Deployment to internal or external platforms.
+4. Make their projects in the VCS open by default.
+
+## Coherent set of actions
+
+The following are actions that need to be performed in order to make the Version Control Systems operational.
+
+<table>
+  <tr>
+    <th>Outcome</th>
+    <th>Action</th>
+    <th>Description</th>
+    <th>Lead</th>
+    <th>Contributor(s)</th>
+  </tr>
+  <tr>
+    <td rowspan="3"><b>Architecture</b></td>
+    <td>Update ESDC OSS Management Framework</td>
+    <td>Update ESDC OSS Management Framework to reflect the expected usage and capabilities of each approved VCS</td>
+    <td><b>EA</b></td>
+    <td>
+    DevCoP<br>
+    IT Strategy<br>
+    ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td>Finalize endorsement of ESDC OSS Management Framework</td>
+    <td>Endorse the implementation of the framework</td>
+    <td><b>EA</b></td>
+    <td>
+    Senior Mgmt<br>
+    ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td>Define best practices and standards</td>
+    <td>Define best practices, standards and procedures for using Git-based version control systems</td>
+    <td><b>DevCoP</b></td>
+    <td>
+      EA<br>
+      ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="3"><b>Operations</b></td>
+    <td>GCcode:<br>
+    Provide official support</td>
+    <td>Determine the most appropriate model for providing official support in GCcode with the current owner (SSC)</td>
+    <td><b>Software Design Services (SDS)</b></td>
+    <td>
+      SSC<br>
+      TBS<br>
+      DevCoP<br>
+      EA<br>
+      IT Strategy<br>
+      ?<br>
+    </td>
+  </tr>
+  <tr>
+    <td>GCcode:<br>
+    Organize projects and monitor</td>
+    <td>Organize ESDC projects in GCcode and monitor usage</td>
+    <td><b>Software Design Services (SDS)</b></td>
+    <td>
+      ?
+    </td>
+  </tr>
+  <tr>
+    <td>GCcode:<br>
+    Enable connectivity to the Internet</td>
+    <td>Enable projects in GCcode to deploy to public cloud service providers</td>
+    <td><b>SSC</b></td>
+    <td>
+      Cloud CoE<br>
+      ?
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="2"><b>Development</b></td>
+    <td>GCcode:<br>
+    Enable DevOps capabilities</td>
+    <td>
+    Ensure DevOps capabilities are available in GCcode, such as:
+    <ul>
+      <li>Continuous Integration (CI)</li>
+      <li>Continuous Deployment (CD)</li>
+    </ul>
+    </td>
+    <td><b>SSC</b></td>
+    <td>
+    DevCoP<br>
```

Comment:
DevOps CoE?

Reply:
Do you mean that DevOps CoE is the official name? According to their [website](https://esdc-devcop.github.io), I believe it's DevCoP.

Reply:
The DevOps CoE is not yet created. There's a CoP but we are asking for the creation of the CoE in charge of implementing the TSDM. As such, I'd see them as the leads?

Reply:
Sorry, I forgot about the proposed DevOps CoE, my bad! I'll definitely add them with the mention "(to be created)", thanks!